src/storage/database/transaction.ts

```diff
@@ -159,10 +162,12 @@ export const createTransaction = async (db: SQLiteDatabase, transaction: ITransa
       ?,
       ?,
       ?,
+      ?,
       ?
     )`,
     [
       transaction.date.toString(),
+      transaction.duration,
```

Comment:
Change to this otherwise web version breaks for unknown reason.

```suggestion
      transaction.duration ?? null,
```